tests/test_utilities.py

```diff
@@ -55,7 +55,7 @@ def test_empty_df():
     [
         (b"abcde", "utf-8"),  # straight up ascii is a subset of unicode
         (b"Eyjafjallaj\xc3\xb6kull", "utf-8"),  # actual unicode
-        (b"\xC4pple", "ISO-8859-1"),  # non-unicode, ISO characterset
+        (b"\xC4pple", "WINDOWS-1252"),  # non-unicode, ISO characterset
```

Comment:
This should go away once you rebase onto master.

Reply:
I tried rebase and got an odd exception so I tried to merge the master and it haven't solved it.
Do you mind editing my fork?

Reply:
Never mind I managed to do that :)